### PR TITLE
Add metadata to `cNFT` tokens

### PR DESCRIPTION
**Pull request by AshenI99**
Added `metadata` for `tokens` and `tokens_for_owner` methods in `cNFT`.

### CODE REVIEW

File: as/cNFT/assembly/models/persistent_tokens_metadata.ts [hunk 1]
```diff
@@ -59,6 +59,21 @@ export class PeristentTokenMetadata {
 
         return tokenMetadata
     }
+
+    get_all(start: i32, end: i32): TokenMetadata[] {
```
Comment (achrafismyname) [inline as/cNFT/assembly/models/persistent_tokens_metadata.ts:63]: I suggest calling this `get`

File: as/cNFT/assembly/models/persistent_tokens_metadata.ts [hunk 2]
```diff
@@ -59,6 +59,21 @@ export class PeristentTokenMetadata {
...
+        let metadataCollection : TokenMetadata[] = [];
+
+        for (let i = 0; i < entries.length; i++) {
+            metadataCollection.push(entries[i].value)
+        }
+
+        return metadataCollection;
+    }
+
+    get_by_id(tokenId: TokenId): TokenMetadata | null {
```
Comment (achrafismyname) [inline as/cNFT/assembly/models/persistent_tokens_metadata.ts:74]: And this `get_for_token`

**Review by achrafismyname — COMMENTED** [review]
Seems good!

File: as/cNFT/assembly/models/persistent_tokens_metadata.ts (hunk 1 above)
Comment (AshenI99) [reply as/cNFT/assembly/models/persistent_tokens_metadata.ts:63]: Sure. :-)

**Pull request closed without merging**